### DISCUSSION TITLE
Ant 5 Beta - Button background color changes to transparent on state change

### DISCUSSION THREAD

**Discussion by lauridskern** [ant-design/ant-design#38532] [Q&A]
Hi! 
I was trying out the new beta and encountered the following issue: the button background color changes to "transparent".

![grafik](https://user-images.githubusercontent.com/8788856/201481389-afa99779-cb04-45dc-8733-869d37756998.png)
![grafik](https://user-images.githubusercontent.com/8788856/201481394-916fce57-1342-4ec0-a7de-25946c4334cf.png)

Reloading fixes this. I am not sure why and how this happens but it happens to all my buttons.

**Comment by zombieJ** [comment]
Could you help to create an issue with reproduce instead?
